Source/Core/VideoCommon/CMakeLists.txt

```diff
@@ -168,6 +168,7 @@ PRIVATE
   spng
   xxhash
   imgui
+  implot
```

Comment:
Changes to add `implot` as a dependency should be done in the PR that is actually using them